src/app/navigation/utils.ts

```diff
@@ -1,5 +1,6 @@
 import { flatten, unflatten } from 'flat';
 import { Action, Location } from 'history';
+import { isNull, omitBy } from 'lodash';
```

Comment:
when using lodash functions please import them one by one like this:
```
import isNull from 'lodash/isNull';
import omitBy from 'lodash/omitBy';
```

this reduces the bundle size.

Reply:
I see:) Do you want me to change it?

Reply:
if you want to search the project and make another pr to fix any places that are doing this, it looks like at least one more in `Dropdown.tsx`

Reply:
So should I create a special card for it?

Reply:
sure, you can make a card in openstax/unified called like `fix imports to reduce bundle size` or something